content/generic.rst

```diff
@@ -21,6 +21,9 @@ something is wrong.
 
 * Rule #3: If it ain't broke, don't fix it!
 
+* Ammendment to Rule #3: If it's not totally broken but still wastes a lot of your time, fix it! 
```

Comment:
not sure if this is rule #3.
i think that this should be a standalone rule ... as not totally broken... is still broken :)